apps/zbugs/shared/queries.ts

```diff
@@ -112,24 +112,199 @@ export const queries = queriesWithContext({
     > | null,
     dir: 'next' | 'prev',
   ) =>
-    applyIssuePermissions(
-      buildListQuery(listContext, issue, dir).one(),
+    buildBaseListQuery(
+      builder.issue,
+      listContext,
+      issue,
+      dir,
       auth?.role,
-    ),
+    ).one(),
+
+  prevNextForAssignee: (
```

Comment:
Realistically these wouldn't be in `shared`, correct? Since they are only needed on the server.